gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -965,8 +967,19 @@ gst_tensor_filter_install_properties (GObjectClass * gobject_class)
       g_param_spec_string ("output-combination", "output tensor(s) combination",
           "Select the output tensor(s) from the input tensor(s) and/or model output",
           "", G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
+  g_object_class_install_property (gobject_class, PROP_SHARING_KEY,
+      g_param_spec_string ("sharing-key",
+          "The key(name) of sharing model representation",
+          "The Name of sharing model representation. "
+          "In case of tensorflow-lite framework, the model representation is interpreter",
```

Comment:
I cannot understand what is sharing key with this.
Is this a term you've defined? or a well-known term (please add a URL if so)?

Reply:
I agree it is bit hard to understand.
The term is not a standard or well-known. I will search the better term.

Furthermore, to help understanding, the additional explanation will be added at [tensor_filter readme](https://github.com/nnstreamer/nnstreamer/blob/main/gst/nnstreamer/tensor_filter/README.md) and linked here.